cadl-tests/src/main/java/com/cadl/longrunning/models/OperationStatusResourceResource.java

```diff
@@ -9,13 +9,13 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/** The OperationStatusResourceResource model. */
+/** Provides status details for long running operations. */
 @Immutable
 public final class OperationStatusResourceResource {
     /*
      * The unique ID of the operation.
      */
-    @JsonProperty(value = "id", required = true)
+    @JsonProperty(value = "id", required = true, access = JsonProperty.Access.WRITE_ONLY)
     private String id;
```

Comment:
This might not be correct. 

Definition from cadl:
```
  @key("operationId")
  @doc("The unique ID of the operation.")
  id: string;
```

Should this field be generated as:
```java
@JsonProperty(value = "operationId", required = true, access = JsonProperty.Access.WRITE_ONLY)
private String id;
```

Reply:
openapi3 gives us "id" so I assume it is fine. (the "operationId" seems on path, not model)
https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAY2FkbC1sYW5nL3ZlcnNpb25pbmciOwrJIGF6dXJlLXRvb2xzL8UsxhFjb3JlIjsKCkBzZXJ2aWNlVGl0bGUoIlRleHQgYXV0aG9yxEgpCi8vQFbJWC7HY2VkKFPGN8cccykKbmFtZXNwYWNlIMREQchDOwoKZW51bSDPMCB7CiAgdjE6ICIyMDIxLTDEAyIsxBQyxhQyLTA5LTEyIiwKfQoKQENhZGwuUmVzdC5yZXNvdXJjZSgicHJvamVjdHMiKQptb2RlbCBQxhHFW0BrZXnEB3Zpc2liaWxpdHkoInJlYWQiKQogIGlkOiBzdOYAniAgZGVzY3JpcHRpb24%2FyhgKICDkANLLEn0KCmludGVyZuQA5OkA4MVuY3JlYXRl5wCAU3RhdHVzTW9uaXRvciBpcyBDdXN0b21Db3JlLlBvbGxpbmdPcGVyYcRxPMc0PsVxQEHkAZsuxSpwzyooyXIu2m7kAN3NHsR%2Fy1VMb25nUnVu5AHLUucBSEPFL1dpdGjnAaJQcm92aWRlZE5hbWXuAJ9kZWxl2FrIT0TFJMs4ICBsaXN0xxBz1zZMaXN0zTRkZXBsb3nea0Fj5QFCCiAgIOgB%2B%2BQCQSDlAZLEAXNsb3TkAMbsAe0gIH3GJccyCiAgxXFnZegApNduUmVhZOsAo30K6wLx6gHhxXzrAq5wYXJlbnTIRChU5AGh9QLNb%2BgB6OQCzyAg5gLR6QH95gHlyEc8VD7vAKVGb3VuZMY%2FLs815wEO5wMQICDJaUntAwV95QKA5QCaSHR0cC5yb3V08gCXb3Ag8QLCVOYA%2FvAAk%2B0BOPkAycc85QFR